frank-doc-doclet/src/main/java/org/frankframework/frankdoc/model/ConfigChild.java

```diff
@@ -49,7 +49,7 @@ public abstract class ConfigChild extends ElementChild {
 	private @Getter(AccessLevel.PACKAGE) String methodName;
 
 	ConfigChild(FrankElement owningElement, FrankMethod method) {
-		super(owningElement);
+		super(owningElement, false);
```

Comment:
Kan je niet beter op de zelfde manier hier kijken of de annotatie er op staat? (net zoals documented en deprecated)

Reply:
Ik had de unsafe attribute op de superclass gedaan, omdat de function signature die superclass als argument kreeg terwijl het ook gewoon de `FrankAttribute` class had kunnen zijn. Aangepast!